app/experimenter/static/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -53,7 +53,8 @@ const ExperimentDetails: React.FC = () => {
       <LabelledRow label="Experiment Owner" value={data.owner} />
       <LabelledRow label="Public Name" value={data.name}>
         <div className="my-2">
-          Bugzilla ticket can be found <a href="#">here</a>.
+          Bugzilla ticket can be found{" "}
+          <a href={data.bugzilla_url ? data.bugzilla_url : "#"}>here</a>.
```

Comment:
Oh I was gonna say just hide the text and link altogether if the url isn't present

Reply:
will do